### PR TITLE
[thread-cert] refactor case 7.1.5 using pktverfiy

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py [hunk 1]
```diff
@@ -77,56 +109,148 @@ def test(self):
...
+            must_next()
+        pkts.filter_wpan_src64(ROUTER).\
+            filter_wpan_dst64(MED).\
+            filter_mle_cmd(MLE_CHILD_UPDATE_RESPONSE).\
+            filter(lambda p: {
+                              SOURCE_ADDRESS_TLV,
+                              MODE_TLV,
+                              ADDRESS_REGISTRATION_TLV
+                             } < set(p.mle.tlv.type) and\
+                   set(p.mle.tlv.addr_reg_iid) < set(_pkt.mle.tlv.addr_reg_iid)
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py:195]: verify the addresses that should be echoed back?

File: tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py [hunk 2]
```diff
@@ -77,56 +109,148 @@ def test(self):
...
+            must_next()
+        pkts.filter_wpan_src64(ROUTER).\
+            filter_wpan_dst64(SED).\
+            filter_mle_cmd(MLE_CHILD_UPDATE_RESPONSE).\
+            filter(lambda p: {
+                              SOURCE_ADDRESS_TLV,
+                              MODE_TLV,
+                              ADDRESS_REGISTRATION_TLV
+                             } < set(p.mle.tlv.type) and\
+                   set(p.mle.tlv.addr_reg_iid) < set(_pkt.mle.tlv.addr_reg_iid)
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py:251]: Verify the addresses that should be echoed back?

File: tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py [hunk 3]
```diff
@@ -77,56 +110,150 @@ def test(self):
...
+            must_next()
+        pkts.filter_wpan_src64(ROUTER).\
+            filter_wpan_dst64(SED).\
+            filter_mle_cmd(MLE_CHILD_UPDATE_RESPONSE).\
+            filter(lambda p: {
+                              SOURCE_ADDRESS_TLV,
+                              MODE_TLV,
+                              ADDRESS_REGISTRATION_TLV
+                             } < set(p.mle.tlv.type) and\
+                   p.mle.tlv.addr_reg_iid is not nullField and\
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py:253]:
```suggestion
                    len(p.mle.tlv.addr_reg_iid) > 0 and\
```

File: tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py [hunk 4]
```diff
@@ -77,56 +110,150 @@ def test(self):
...
+            must_next()
+        pkts.filter_wpan_src64(ROUTER).\
+            filter_wpan_dst64(MED).\
+            filter_mle_cmd(MLE_CHILD_UPDATE_RESPONSE).\
+            filter(lambda p: {
+                              SOURCE_ADDRESS_TLV,
+                              MODE_TLV,
+                              ADDRESS_REGISTRATION_TLV
+                             } < set(p.mle.tlv.type) and\
+                   p.mle.tlv.addr_reg_iid is not nullField and\
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py:196]:
```suggestion
                   len(p.mle.tlv.addr_reg_iid) > 0 and\
```

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1: 

with two nits

**Review by jwhui — APPROVED** [review]

**Pull request merged**